src/html-conventions-lib/class-attributes-with-same-name.xsl

```diff
@@ -112,13 +112,13 @@
                     ()
                 else
                     if (fn:boolean($definitionValues)) then
-                        f:generateHtmlWarning(fn:concat('When a property is reused in multiple contexts, the meaning given by the definition is expected to be the same. ',
-                        'In this case, multiple definitions are found: ',
-                        fn:string-join($descriptionsWithAnnotations, ',')))
+                    f:generateFormattedHtmlWarning(fn:concat('When a property is reused in multiple contexts, the meaning given by the definition is expected to be the same. ',
```

Comment:
wonderful solution. We need to discuss it, however. 
I wonder what the usage cases are (single and multiple values, i.e. a sequence) and whether to switch to HTML message in the case of single values.